src/App.tsx

```diff
@@ -0,0 +1,59 @@
+import { SearchBar } from "./SearchBar";
+import ResultsTable from "./ResultsTable";
+import data from "./data.json";
+import { useState } from "react";
+import Tabs from "./Tabs";
+
+export default function App() {
```

Comment:
The app is usually simple component, it should not do much. However, it should render other components. I would recommend that you extract the parking relocated logic and rendering in different components. It's fine as it is for now though.